[2024-09-08T18:48:41.801+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-08T18:48:41.829+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_dag_v2.task1 backfill__2024-09-01T00:00:00+00:00 [queued]>
[2024-09-08T18:48:41.849+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_dag_v2.task1 backfill__2024-09-01T00:00:00+00:00 [queued]>
[2024-09-08T18:48:41.852+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 4
[2024-09-08T18:48:41.905+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): task1> on 2024-09-01 00:00:00+00:00
[2024-09-08T18:48:41.923+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2415) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-08T18:48:41.933+0000] {standard_task_runner.py:72} INFO - Started process 2422 to run task
[2024-09-08T18:48:42.005+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'catchup_dag_v2', 'task1', 'backfill__2024-09-01T00:00:00+00:00', '--job-id', '261', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpp8jo_15p']
[2024-09-08T18:48:42.015+0000] {standard_task_runner.py:105} INFO - Job 261: Subtask task1
[2024-09-08T18:48:42.856+0000] {task_command.py:467} INFO - Running <TaskInstance: catchup_dag_v2.task1 backfill__2024-09-01T00:00:00+00:00 [running]> on host 2abf3f0b0579
[2024-09-08T18:48:43.240+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='catchup_dag_v2' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-09-01T00:00:00+00:00'
[2024-09-08T18:48:43.252+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-08T18:48:43.257+0000] {logging_mixin.py:190} INFO - hello
[2024-09-08T18:48:43.258+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-08T18:48:43.276+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-08T18:48:43.278+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=catchup_dag_v2, task_id=task1, run_id=backfill__2024-09-01T00:00:00+00:00, execution_date=20240901T000000, start_date=20240908T184841, end_date=20240908T184843
[2024-09-08T18:48:43.372+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-08T18:48:43.469+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-08T18:48:43.472+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
